### DISCUSSION TITLE
Any plan to upgrade linux kernel to 6.12

### DISCUSSION THREAD

**Discussion by RBEI-VENKAT** [polarfire-soc/polarfire-soc-documentation#505] [Linux build systems for PolarFire SoC] [answered]
Hello,
Is there any plan to upgrade Linux kernel v6.12 once it released, which supports PREEMPT_RT for RISC-V also?

**Comment by vfalanis** [comment, marked as answer]
Hello @RBEI-VENKAT,

We are planning to upgrade the Linux4Microchip kernel to version 6.12 around the end of the second quarter of 2025
